USAGE

```diff
@@ -13,6 +13,7 @@ DESCRIPTION
 
                             [root]
                             ├── S1
+                            │   ├── *lesion_mask.nii.gz (optional)
```

Comment:
Please add *lesion_mask.nii.gz (optional) in S2 as well below

Reply:
Also, can you please add 1-2 sentences in the first USAGE text "Compute tractometry on each bundles/metrics" to describe the optional lesion_mask option. What does it do, what does it change in the outputs. Thanks

Reply:
Also, should the --min_lesion_vol be given as an option to the user somewhere?

Reply:
@mdesco It is now a parameter